docs-preview/README.md

```diff
@@ -1,15 +1,10 @@
-# markdocs
-[![Current Version](https://vsmarketplacebadge.apphb.com/version/qezhu.markdocs.svg)](https://marketplace.visualstudio.com/items?itemName=qezhu.markdocs)
+# Docs Preview Extension
 
-Markdown preview tool that supports docfx flavored markdown and CommonMark syntax.
-
-## Install
-
-Press `F1`, type `ext install markdocs`
+Markdown preview tool for CommonMark syntax and markdown extension syntax supported in Docs.
 
 ## Usage
 
-<img src="https://github.com/qinezh/vscode-markdocs/raw/master/images/howto.gif" width="600"/>
+<img src="https://github.com/Microsoft/vscode-docs-authoring/raw/master/images/howto.gif" width="600"/>
```

Comment:
Hi @qinezh , I think this path may need to change due to the repo structure.  We can either add the images to the root of the repo or specify the docs-preview folder path i.e. https://github.com/Microsoft/vscode-docs-authoring/tree/master/docs-preview/images. Thx

Reply:
@jamarw @qinezh @meganbradley we need to update the image to if we're going to keep it. It's using the name "Markdocs preview"

Reply:
Agree. The gif is removed and we could update and add it back once the extension is published.